### PR TITLE
Updated changelog and readme

### PR DESCRIPTION
**Pull request by kushagraThapar**
Updated changelog with all releases. 
Updated readme with new version and reactor Schedulers.

### CODE REVIEW

**Comment by kushagraThapar** [comment]
> is `com.microsoft.azure` change a change across all sdks?

I think yes. These are the new documentation for v4 track-2 libraries: https://azure.github.io/azure-sdk-for-java/track2reports/

They all have com.azure.* packages

File: sdk/cosmos/README.md [hunk 1]
```diff
@@ -151,16 +151,15 @@ readItemMono
...
 
 readItemMono
-  .subscribeOn(Schedulers.computation())
+  .subscribeOn(Schedulers.parallel())
   .subscribe(
   resourceResponse -> {
-    // this is executed on threads provided by Scheduler.computation()
-    // Schedulers.computation() should be used only the work is cpu intensive and you are not doing blocking IO, thread sleep, etc. in this thread against other resources.
+    // this is executed on threads provided by Scheduler.parallel()
+    // Schedulers.parallel() should be used only the work is cpu intensive and you are not doing blocking IO, thread sleep, etc. in this thread against other resources.
```
Comment (mbhaskar) [inline sdk/cosmos/README.md]: Minor suggestion: ...used when/if the work is cpu intensive..

**Pull request merged**